src/test/java/io/strimzi/test/container/KafkaNodeRoleTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.container;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class KafkaNodeRoleTest {
+
+    @Test
+    void testMixedRoleProcessRoles() {
+        assertThat(KafkaNodeRole.COMBINED.getProcessRoles(), is("broker,controller"));
+    }
+
+    @Test
+    void testControllerOnlyProcessRoles() {
+        assertThat(KafkaNodeRole.CONTROLLER.getProcessRoles(), is("controller"));
+    }
+
+    @Test
+    void testBrokerOnlyProcessRoles() {
+        assertThat(KafkaNodeRole.BROKER.getProcessRoles(), is("broker"));
+    }
+
+    @Test
+    void testMixedRoleIsController() {
+        assertThat(KafkaNodeRole.COMBINED.isController(), is(true));
+    }
+
+    @Test
+    void testControllerOnlyIsController() {
+        assertThat(KafkaNodeRole.CONTROLLER.isController(), is(true));
+    }
+
+    @Test
+    void testBrokerOnlyIsController() {
+        assertThat(KafkaNodeRole.BROKER.isController(), is(false));
+    }
+
+    @Test
+    void testMixedRoleIsBroker() {
+        assertThat(KafkaNodeRole.COMBINED.isBroker(), is(true));
+    }
+
+    @Test
+    void testControllerOnlyIsBroker() {
```

Comment:
We renamed the enum value, should we also update the test names. Same in the few other places in this file.